prefab/curl-ssl/README.md

```diff
@@ -4,9 +4,24 @@ The curl-ssl sample shows how to import native dependencies from Maven. This
 sample uses [curl], [OpenSSL], and [JsonCpp] to display a list of the most
 recent 10 reviews submitted to AOSP's code review system.
 
-[curl]: https://curl.haxx.se/
-[OpenSSL]: https://www.openssl.org/
+To use [Prefab] with Android Gradle Plugin 4.0+, simply follow the following 3 steps:
+
+* Turn on prefab feature in [gradle.properties]
+* Declare the dependencies in the app's module [build.gradle]
+* Add the dependencies into the native module's [android.mk] or [CMakeLists.txt]
+
+From here on, the headers and native libs in the dependencies are [ready to use]!
+
+[Prefab]:https://github.com/google/prefab
```

Comment:
Use https://google.github.io/prefab instead. It's a better landing page and it does link to the repo.

Reply:
it is better landing page, thanks!